.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,9 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Report a bug/issues with an unsupported Supervisor
+    url: https://community.home-assistant.io/c/community-guides
```

Hmm, people should not report a bug with the community guides section (as it is a wiki).

Maybe just point them to the forums in general:

```suggestion
    url: https://community.home-assistant.io
```